src/game/behaviors/star_door.inc.c

```diff
@@ -61,4 +61,10 @@ void bhv_star_door_loop(void) {
             o->oAction = STAR_DOOR_ACT_CLOSED;
             break;
     }
+
+    if (o->oAction != STAR_DOOR_ACT_OPEN) {
+        load_object_collision_model();
+    }
+
+    bhv_door_rendering_loop();
```

Comment:
please revert this (and the bhv script) since its unrelated to the bug fix, different file altogether, plus `bhv_*_loop` functions are typically called from the behavior script anyways. and im not sure why they decided to load the collision model at all times but most collision models are loaded through behavior scripts anyways, so i dont want to question the original logic here right now, not worth it when the jam starts tomorrow

Reply:
It's not unrelated. In vanilla whatever `GRAPH_RENDER_ACTIVE` got set to by `bhv_door_rendering_loop` would get overwritten by the drawing distance checks in `load_object_collision_model`, causing star doors to render outside of rooms. They were swapped so that the room check takes priority and moved to the behavior file because that's where they are in door.inc.c. I'll revert the `STAR_DOOR_ACT_OPEN` check.